railties/lib/rails/generators/migration.rb

```diff
@@ -63,7 +63,12 @@ def migration_template(source, destination, config = {})
         numbered_destination = File.join(dir, ["%migration_number%", base].join("_"))
 
         create_migration numbered_destination, nil, config do
-          ERB.new(::File.binread(source), nil, "-", "@output_buffer").result(context)
+          match = ERB.version.match(/\Aerb\.rb \[(?<version>[^ ]+) /)
+          if match && match[:version] >= '2.2.0' # Ruby 2.6+
```

Comment:
Use double quotes.

![image](https://user-images.githubusercontent.com/12642/36968286-7f2662f2-20a5-11e8-93d1-4254beab6b71.png)

Reply:
I made a basic error 💦 I fixed it.